### PR TITLE
Update ILogger XML documentation

### PR DESCRIPTION
**Pull request by erichiller**
- Update ILogger examples XML documentation to use `<code>`
- Fix an errant `>`

### CODE REVIEW

**Comment by nblumhardt** [comment]
Thanks!

File: src/Serilog/ILogger.cs [hunk 1]
```diff
@@ -398,7 +404,10 @@ void Verbose<T>(string messageTemplate, T propertyValue)
         /// <param name="propertyValue0">Object positionally formatted into the message template.</param>
         /// <param name="propertyValue1">Object positionally formatted into the message template.</param>
         /// <example>
+        /// <code>
         /// Log.Verbose("Staring into space, wondering if we're alone.");
+        /// <code>
+        /// <code>
```
Comment (jstclair) [inline src/Serilog/ILogger.cs:410]: double-closing tag

File: src/Serilog/ILogger.cs [hunk 2]
```diff
@@ -1148,7 +1249,11 @@ void Error<T>(Exception? exception, string messageTemplate, T propertyValue)
         /// <param name="messageTemplate">Message template describing the event.</param>
         /// <param name="propertyValues">Objects positionally formatted into the message template.</param>
         /// <example>
+        /// <code>
+        /// <code>
```
Comment (jstclair) [inline src/Serilog/ILogger.cs:1253]: double opening tags

File: src/Serilog/ILogger.cs [hunk 3]
```diff
@@ -1148,7 +1249,11 @@ void Error<T>(Exception? exception, string messageTemplate, T propertyValue)
...
         /// Log.Error(ex, "Failed {ErrorCount} records.", brokenRecords.Length);
+        /// </code>
+        /// <code>
```
Comment (jstclair) [inline src/Serilog/ILogger.cs:1256]: broken open tag after closing tag

**Comment by jstclair** [comment]
I saw you reverted this PR @nblumhardt  but I think those issues above are the cause - do you want to fix up this small issues @erichiller ?

**Comment by nblumhardt** [comment]
These are fixed now on dev, @jstclair - thanks for the note, though!

**Pull request merged**